core/src/main/java/io/servicecomb/core/Const.java

```diff
@@ -34,5 +34,5 @@ private Const() {
 
     public static final String SRC_MICROSERVICE = "x-cse-src-microservice";
 
-    public static final String DEST_MICROSERVICE = "x-cse-dest-microservice";
+    public static final String TARGET_MICROSERVICE = "x-cse-target-microservice";
```

Comment:
If the header is used internally, it's OK to change the header name, but if it is used from outside, we should keep the old header there.